src/fides/core/exceptions.py

```diff
@@ -0,0 +1,10 @@
+from rich_click import secho
+
+
+class DockerCheckException(Exception):
+    """Raise an error when `local_mode` is enabled with an incompatible command."""
```

Comment:
nit - looks like a copy pasta miss

Reply:
I've been found out 😂  good catch, thank you!